config/core/build-configs-stable.yaml

```diff
@@ -1,5 +1,8 @@
 trees:
 
+  cip:
+    url: "https://git.kernel.org/pub/scm/linux/kernel/git/cip/linux-cip.git"
```

Comment:
I don't know if this will cause any issues because we already define this in [build-configs-cip.yaml](https://github.com/kernelci/kernelci-core/blob/main/config/core/build-configs-cip.yaml#L2)

Reply:
I don't think this is a problem as long as you get the right tree.

Reply:
What's the point of putting CIP trees in the stable build configs though?  I think this should really have been in `build-configs-cip.yaml` instead.

Reply:
Hi @gctucker 

My thinking was that the linux-4.4.y-st branch is basically a replacement for linux-4.4.y LTS.

If you'd rather we move these branches to configs-cip I can, let me know.

Chris